### PR TITLE
[Config][Lock][SecurityBundle]  do not use uniqid() in tests

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 7.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | part of #57588
| License       | MIT

### CODE REVIEW

**Review by derrabus — APPROVED** [review]

File: src/Symfony/Component/Config/Tests/Resource/ReflectionClassResourceTest.php [hunk 1]
```diff
@@ -117,49 +117,49 @@ public function testHashedSignature(bool $changeExpected, int $changedLine, ?str
...
+        yield [true, 11, 'public function pub($arg = null): Foo {}', '21'];
+        yield [false, 11, "public function pub(\$arg = null) {\nreturn 123;\n}", '22'];
+        yield [true, 12, '/** prot docblock */', '23'];
+        yield [true, 13, 'protected function prot($a = [123]) {}', '24'];
+        yield [true, 13, '#[Foo] protected function prot($a = []) {}', '25'];
+        yield [true, 13, 'protected function prot(#[Foo] $a = []) {}', '26'];
+        yield [true, 13, '#[Foo(new MissingClass)] protected function prot($a = []) {}', '27'];
+        yield [true, 13, 'protected function prot(#[Foo(new MissingClass)] $a = []) {}', '28'];
+        yield [false, 14, '/** priv docblock */', '29'];
+        yield [false, 15, null, '30'];
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Config/Tests/Resource/ReflectionClassResourceTest.php]: worth using `++$i` to make it easier to maintain?

Comment (GromNaN) [reply src/Symfony/Component/Config/Tests/Resource/ReflectionClassResourceTest.php]: Or a hash of [`TestCase::nameWithDataSet`](https://github.com/sebastianbergmann/phpunit/blob/0441c6da39bd9dc477206ae0591b678c7e9da0aa/src/Framework/TestCase.php#L396C27-L396C42)

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @xabbuh.

File: src/Symfony/Component/Config/Tests/Resource/ReflectionClassResourceTest.php (hunk 1 above)
Comment (fabpot) [reply src/Symfony/Component/Config/Tests/Resource/ReflectionClassResourceTest.php]: Done while merging

**Pull request merged**